global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "8.0.100-alpha.1.23061.8",
+    "dotnet": "7.0.101",
```

Comment:
Do we need to revert any of the arcade changes? Can't we just have the two changes in this file that lock tools and sdk dotnet version to "7.0.101" and revert everything else in the PR?

Reply:
Maybe the suggestion above might not work. I think someone from the arcade infra team should take a look.

Reply:
Yeah I tried to only update global.json and that didn't work.

```
.packages/microsoft.dotnet.arcade.sdk/8.0.0-beta.23066.1/tools/SourceBuild/SourceBuildArcadeTools.targets(20,5): error MSB4018: (NETCORE_ENGINEERING_TELEMETRY=Restore) The "ReadSourceBuildIntermediateNupkgDependencies" task failed unexpectedly.
System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.

```